core/src/main/scala/c4s/process/Extract.scala

```diff
@@ -0,0 +1,17 @@
+package c4s.process
+
+import cats.effect.IO
+
+trait Extract[F[_]] {
+  def extract[A](fa: F[A]): A
```

Comment:
Maybe to mark that this is a _side effect_ method
```scala 
   def extract[A](fa: F[A]): Either[Throwable, A]
```
This can help to log problems in future.

Reply:
Then, How do I handle the `Throwable` when we are [extracting](https://github.com/cats4scala/cats-process/pull/59/files#diff-27831dc61faa522645536d501346d4b8R41) the value?

Reply:
At the moment, I don't think we require to it ... maybe in future we should log these kind of issues.